ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__opentelemetry-product-ocp-4.15-lp-interop.yaml

```diff
@@ -38,18 +38,18 @@ tests:
       EO_SUB_SOURCE: redhat-operators
       FIREWATCH_CONFIG: |
         [
-          {"step": "build-logs", "failure_type": "pod_failure", "classification": "OpenTelemetry operator images", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
+          {"step": "build-logs", "failure_type": "pod_failure", "classification": "OpenTelemetry operator images", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
           {"step": "build-logs", "failure_type": "test_failure", "classification": "OpenTelemetry operator images", "jira_project": "TRACING", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
           {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure provisioning", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp"]},
-          {"step": "install", "failure_type": "pod_failure", "classification": "OpenTelemetry operator install", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "install", "failure_type": "test_failure", "classification": "OpenTelemetry operator install", "jira_project": "TRACING", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "distributed-tracing-install-*", "failure_type": "pod_failure", "classification": "Distributed tracing operators", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "distributed-tracing-install-*", "failure_type": "test_failure", "classification": "Distributed tracing operators", "jira_project": "TRACING", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "operatorhub-subscribe-*", "failure_type": "pod_failure", "classification": "Distributed tracing dependency operators", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
-          {"step": "operatorhub-subscribe-*", "failure_type": "test_failure", "classification": "Distributed tracing dependency operators", "jira_project": "TRACING", "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
+          {"step": "install", "failure_type": "pod_failure", "classification": "OpenTelemetry operator install", "jira_project": "LPINTEROP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","opentelemetry-lp","interop-tests"]},
```

Comment:
I also don't see the step named install that is run by this scenario

It seems to me this rule will never take effect.

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-opentelemetry-product-ocp-4.15-lp-interop-opentelemetry-interop-aws/1723944624017379328/artifacts/opentelemetry-interop-aws/